Summary/analyze/MethodLinker/python_matcher.py

```diff
@@ -277,3 +282,46 @@ def python_match_examples(repo_name, examples, functions, classes):
                 seen.add((link[0], link[1]))
 
     return method_calls
+
+
+def get_declared_variable_mapping(example_code, classes):
```

Comment:
maybe I missed it but where is this function called/used?

Reply:
It's not used anywhere in this PR. It is used in the next PR #15